core/net/src/central/mod.rs

```diff
@@ -1,11 +1,8 @@
 mod api;
-#[cfg(any(feature = "service", test))]
```

Comment:
Modules gated by the `service` feature are not needed  by `core::{activity, market, payment, vpn}` and `ya-transfer` (imported by ExeUnit)